test/ApacheOrcDotNet.Test/ColumnTypes/BooleanColumn_Test.cs

```diff
@@ -14,10 +14,11 @@ public class BooleanColumn_Test
 		[Fact]
 		public void RoundTrip_BooleanColumn()
 		{
-			RoundTripSingleBool(70000);
+			RoundTripSingleBool(70000, 1000);
+			RoundTripSingleBool(70000, 10);
```

Comment:
Let's keep the original test, and add a new one, for the problematic use case.

Reply:
Good point. I've restored the original test case with a rowIndexStride of 10000.